packages/v4/patternfly-docs/content/accessibility/banner/banner.md

```diff
@@ -3,11 +3,42 @@ id: Banner
 section: components
 ---
 
-**Keyboard users** should not be able to focus on a banner, as it is not an interactive element.
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
 
-**Screen reader users** should be able to have a screen reader describe the contents of a banner, but not focus on or interact with a banner otherwise.
+## Accessibility
+
+To implement an accessible PatternFly **banner**:
 
-## Accessibility application:
 - Check the color contrast of a banner and any text on it to make sure they can be read by users with color blindness or other visual impairments.
+- When a banner is used to convey status, add an icon that also conveys status visually, and provide screen reader text using `.pf-screen-reader` or an equivalent.
+
+## Testing
+
+At a minimum, an banner should meet the following criteria:
+
+<List isPlain>
+  <ListItem>
+    <Checkbox id="banner-a11y-checkbox-1" label="A user should not be able to focus on the banner with keyboard controls." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="banner-a11y-checkbox-2" label="A user should be able to have a screen reader describe the contents of the banner, but not focus on or interact with the banner otherwise." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="banner-a11y-checkbox-3" label={<span>If the banner conveys status, a status icon and <code className="ws-code">.pf-screen-reader</code> element is included.</span>} />
```

Comment:
Bummer we can't use markdown in react props like this. I noticed this format in the accordion docs, too. I'm guessing there isn't a way around it? Passing the prop values as children is the only way I have seen how to do it. Just seems like it will lead to inconsistencies.

https://github.com/patternfly/patternfly-org/blob/42a6910f36c412a428852947599154935f0ff1ef/packages/v4/patternfly-docs/content/accessibility/accordion/accordion.md#L26-L42